Lib/smtplib.py

```diff
@@ -958,7 +958,7 @@ def send_message(self, msg, from_addr=None, to_addrs=None,
             if international:
                 g = email.generator.BytesGenerator(
                     bytesmsg, policy=msg.policy.clone(utf8=True))
-                mail_options += ['SMTPUTF8', 'BODY=8BITMIME']
+                mail_options = (*mail_options, 'SMTPUTF8', 'BODY=8BITMIME')
```

Comment:
I don't like this change.  `+=` seems more common/readable, and it's valid for immutable objects
like tuple and str.  (You need to change `[]` to `()` in this case)

```
>>> x = (1,2)
>>> x += (3,4)
>>> x
(1, 2, 3, 4)
```